fundamentals/authors.tex

```diff
@@ -6,7 +6,5 @@
   Matthew J. Gidden$^2$,\\
   Meghan B. McGarry$^2$,\\
   Arrielle C. Opotowsky$^2$,\\
-  Olzhas Rakhimov$^2$,\\
-  Zach Welch$^2$,\\
```

Comment:
As long as they are unresponsive, then it is appropriate to move them to the acknowledgements.